src/main/java/uk/gov/hmcts/reform/sscs/service/servicebus/TribunalsEventQueueListenerService.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.sscs.service.servicebus;
+
+import com.azure.core.amqp.exception.AmqpException;
+import com.azure.messaging.servicebus.ServiceBusReceiverClient;
+import com.azure.messaging.servicebus.ServiceBusSessionReceiverClient;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingState;
+import uk.gov.hmcts.reform.sscs.model.hearings.HearingRequest;
+import uk.gov.hmcts.reform.sscs.service.HearingsService;
+
+@Service
+@Slf4j
+public class TribunalsEventQueueListenerService {
+
+    private HearingsService hearingsService;
+
+    public void processMessage(ServiceBusSessionReceiverClient receiverClient) {
```

Comment:
minor: try block in a try block is.. interesting, can you extract the inner try to a separate method? Can be in the same class.